src/Kernel-Tests-Extended/AsciiCharsetTest.class.st

```diff
@@ -28,15 +28,16 @@ AsciiCharsetTest >> testIsCasedLetter [
 
 { #category : #'generated tests' }
 AsciiCharsetTest >> testIsClosePunctuation [
-
 	"Exhaustively check that the charset under test behaves like Unicode"
 
 	| charset |
 	charset := self classUnderTest.
 	0 to: charset maxValue do: [ :asciiValue | 
 		| ch |
 		ch := Character value: asciiValue.
-		self assert: (Unicode isClosePunctuation: ch) equals: (charset isClosePunctuation: ch) ]
+		self
```

Comment:
I am not a big fan of all the line returns in those tests. Almost everyone working on Pharo have at least a 13', and those lines are for frow wrapping on a 13'.

Reply:
Then blame the standard formatter.